apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -1008,6 +1008,10 @@ export function getSectionRows(state, sectionIds) {
   }));
 }
 
+export function getAssignmentName(state, sectionId) {
+  const {sections, validAssignments} = getRoot(state);
+  return assignmentNames(validAssignments, sections[sectionId])[0];
```

Comment:
What does it mean if assignmentNames returns multiple values? How are you sure that its safe to take the first thing returned by the assignmentNames function and use it as our assignment name, rather than one of the other values?

Reply:
Sections can currently only have one assignment; this was built on the assumption that in the future there might be multiple.